### PR TITLE
Alters ASay chat color back to the old MSay color

### PR DESCRIPTION
**Pull request by blackdragonTOW**
Changing Asay back to the Msay color that many were used to for a very long time.

# Explain why it's good for the game

Because this color doesn't blend in with other major chat colors (such as dsay, and others), and it reminds us of simpler times.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
admin: Adjustment of ASay color
/:cl:

### CODE REVIEW

**Review by realforest2001 — CHANGES_REQUESTED** [review]
This is currently making tickets appear brown too, please can you make it so tickets remain purple so they're easier to identify.

**Comment by blackdragonTOW** [comment]
Removed changes to headminsay colors as those do not alter the asay color of hosts in asay.

Additionally, as it stands before this change, ASay is purple.  Tickets are purple.  There hasn't been confusion about those colors being the same, so the change here is fine.   Tickets are a function of ASay and move through that channel, thus the coloring.
To make tickets and asay separate we would have to add a great deal that would be outside the scope of this color change PR and would need to be atomized with another PR.

**Comment by blackdragonTOW** [comment]
What if we did.........this....

**Comment by blackdragonTOW** [comment]
Okay, this should achieve the following goals:
Headmins have different colors in ASay
ASay chat messages are the OG mod colors
Ahelps are still purple
ASay and AHelps are now distinct colors.

File: code/modules/admin/tabs/admin_tab.dm [hunk 1]
```diff
@@ -255,9 +255,9 @@
 
 	log_adminpm("ADMIN: [key_name(src)] : [msg]")
 
-	var/color = "adminsay"
+	var/color = "mod"
 	if(ishost(usr))
```
Comment (realforest2001) [inline code/modules/admin/tabs/admin_tab.dm:259]:
```suggestion
	if(check_rights(R_PERMISSIONS))
```

**Review by realforest2001 — APPROVED** [review]

File: code/modules/admin/tabs/admin_tab.dm [hunk 2]
```diff
@@ -255,9 +255,9 @@
 
 	log_adminpm("ADMIN: [key_name(src)] : [msg]")
 
-	var/color = "adminsay"
-	if(ishost(usr))
-		color = "headminsay"
+	var/color = "mod"
+	if(check_rights(R_PERMISSIONS))
```
Comment (harryob) [inline code/modules/admin/tabs/admin_tab.dm:259]:
```suggestion
	if(check_rights(R_PERMISSIONS, show_msg = FALSE))
```
or you'll spam non-permissioned users when they try to talk

**Review by harryob — CHANGES_REQUESTED** [review]

**Review by harryob — APPROVED** [review]

**Pull request merged**